src/rules/style/relational_symbols.rs

```diff
@@ -0,0 +1,87 @@
+use std::collections::BTreeMap;
+
+use crate::ast::FortitudeNode;
+use crate::settings::Settings;
+use crate::{ASTRule, Rule, Violation};
+use lazy_static::lazy_static;
+use tree_sitter::Node;
+
+lazy_static! {
+    static ref RELATIONAL_NAMES: BTreeMap<&'static str, &'static str> = {
+        BTreeMap::from([
+            (".gt.", ">"),
+            (".ge.", ">="),
+            (".lt.", "<"),
+            (".le.", "<="),
+            (".eq.", "=="),
+            (".ne.", "/="),
+        ])
+    };
+}
+
+pub struct RelationalSymbol {}
+
+impl Rule for RelationalSymbol {
+    fn new(_settings: &Settings) -> Self {
+        Self {}
+    }
+
+    fn explain(&self) -> &'static str {
+        "
+        Fortran 90 introduced the traditional symbols for relational expressions: `>`,
+        `>=`, `<`, and so on.
+        "
+    }
+}
+
+impl ASTRule for RelationalSymbol {
+    fn check(&self, node: &Node, src: &str) -> Option<Vec<Violation>> {
+        let relation = node.child(1)?;
+        let symbol = relation.to_text(src)?.to_lowercase();
+        let new_symbol = RELATIONAL_NAMES.get(symbol.as_str())?;
```

Comment:
If using the match function I suggested...
```suggestion
        let new_symbol = map_relational_symbols(symbol.as_str())?;
```